### ISSUE TITLE
Village Area

### ISSUE THREAD

**Issue by LunarGem** [LunarGem/fire-and-ice#23]
Finish the village area:

- [ ] Inventory
- [ ] Shop
- [ ] Field
- [ ] Training

**Comment by LunarGem** [comment]
Putting this on hold temporarily cause I don't feel like arguing with the cards just this very instant...

**Comment by LunarGem** [comment]
The inventory requires that everything be a bit more OO so I'm going to start reworking everything (which may cause merge conflicts later on but I'm prepared to deal with those).

**Comment by LunarGem** [comment]
Cards are fixed! Breadcrumbs now arguing with me... 😛 

Dependent on #11

**Comment by LunarGem** [comment]
Breadcrumbs are finished, but the inventory needs to wait for the carousel bug to get fixed.
